data/phi0474/phi014/phi0474.phi014.perseus-eng2.xml

```diff
@@ -43,15 +40,19 @@
    <sourceDesc>
     <biblStruct>
      <monogr>
-      <author>M. Tullius Cicero</author>
-      <title>The Orations of Marcus Tullius Cicero, literally translated by C. D. Yonge, B.
-       A.</title>
+      <author xml:lang="lat">M. Tullius Cicero</author>
+      <title>The Orations of Marcus Tullius Cicero</title>
+      <title type="sub">Three orations on the Agrarian law, the four against Catiline, the orations 
+       for Rabirius, Murena, Sulla, Archias, Flaccus, Scaurus, etc.</title>
+      <editor role="translator">C. D. Yonge</editor>
       <imprint>
        <pubPlace>London</pubPlace>
-       <publisher>Henry G. Bohn, York Street, Covent Garden.</publisher>
+       <publisher>George Bell &amp; Sons</publisher>
```

Comment:
@AlisonBabeu 
This might be an instance where you want to check the physical copy in the office. There is a Yonge/Bohn edition and it's likely that's the imprint we used if that's the one in the office.

Reply:
hey @lcerrato I had made this change in the files previous to these translations as well. I may hold off on changing the rest of these until I get a chance to go into the office (hopefully soon!) Or should I just continue forward and not edit that part of the publisher statement yet do you think?

Reply:
you know @lcerrato looking back I was basing this metadata on the Perseus Catalog, which used a later reprint of this Bohn/Yonge edition that has slightly different publication data even though it has the exact same translations. I'm going to go back through the older English translations that I already edited as part of these Cicero pull requests and add the Henry G. Bohn metadata back at the end and not change it any more at this point.